### PR TITLE
Retrieve the metadata files in da.json

### PR DESCRIPTION
**Pull request by ruhan1**
This is the second part: retrieve metadata.

### CODE REVIEW

File: pkg/common/util.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package common
+
+func Check(e error) {
```
Comment (ligangty) [inline pkg/common/util.go:19]: If you just panic an error in this method, I think rename this method to something like "RePanic" is better.

File: pkg/dataset/run.go [hunk 2]
```diff
@@ -27,15 +27,20 @@ import (
...
+	PncBaseUrl     string `json:"pncBaseUrl"`
+	BuildId        string `json:"buildId"`
+	BuildType      string `json:"buildType"`
+	TemporaryBuild bool   `json:"temporaryBuild"`
 }
 
 const DATASET_DIR = "dataset"
+const DA_JSON = "da.json"
+const TRACKING_JSON = "tracking.json"
+const INFO_JSON = "info.json"
```
Comment (ligangty) [inline pkg/dataset/run.go:43]: A nitpick: we can use multi const declare here:
const(
DATASET_DIR = "dataset"
DA_JSON = "da.json"
TRACKING_JSON = "tracking.json"
INFO_JSON = "info.json"
)

File: pkg/dataset/run.go [hunk 3]
```diff
@@ -83,26 +78,37 @@ func Run(pncBaseUrl, indyBaseUrl, buildId string) {
...
+		buildJsonFileLoc = path.Join(dirLoc, "build.json")
+		if !common.FileOrDirExists(buildJsonFileLoc) {
+			success := common.DownloadFile(buildURL, buildJsonFileLoc)
 			if !success {
 				fmt.Println("Download build.json failed.")
 				return
 			}
-			formatJsonFile(fileLoc)
+		}
+	}
```
Comment (ligangty) [inline pkg/dataset/run.go:101]: Maybe we can extract a function here to avoid these two duplicated if...else... code block?

Comment (ruhan1) [reply pkg/dataset/run.go:101]: Ok.

**Pull request merged**